web/packages/teleport/src/Integrations/DeleteAwsOidcIntegrationDialog.tsx

```diff
@@ -0,0 +1,94 @@
+/**
+ * Teleport
+ * Copyright (C) 2025  Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { Alert, ButtonSecondary, ButtonWarning, P1, Text } from 'design';
+import Dialog, {
+  DialogContent,
+  DialogFooter,
+  DialogHeader,
+  DialogTitle,
+} from 'design/DialogConfirmation';
+import useAttempt from 'shared/hooks/useAttemptNext';
+
+import { IntegrationAwsOidc } from 'teleport/services/integrations';
+import useStickyClusterId from 'teleport/useStickyClusterId';
+
+import { type DeleteRequestOptions } from './Operations/IntegrationOperations';
+
+type Props = {
+  close(): void;
+  remove(opt?: DeleteRequestOptions): Promise<void>;
+  integration: IntegrationAwsOidc;
+};
+
+export function DeleteAwsOidcIntegrationDialog(props: Props) {
+  const { close, remove, integration } = props;
+  const { attempt, run } = useAttempt();
+  const isDisabled = attempt.status === 'processing';
+  const { clusterId } = useStickyClusterId();
+
+  const awsResourceExplorerUrl =
+    `https://resource-explorer.console.aws.amazon.com/resource-explorer/home#/search?query=` +
+    `tag%3Ateleport.dev%2Forigin%3Dintegration_awsoidc+` + // tag:teleport.dev/origin=integration_awsoidc
+    `tag%3Ateleport.dev%2Fcluster%3D${clusterId}+` + // tag:teleport.dev/cluster={cluster_name}
+    `tag%3Ateleport.dev%2Fintegration%3D${integration.name}`; // tag:teleport.dev/integration={integration_name}
+  function onOk() {
+    run(() => remove({ deleteAssociatedResources: true }));
+  }
+
+  return (
+    <Dialog onClose={close} open={true}>
+      <DialogHeader>
+        <DialogTitle>Delete Integration?</DialogTitle>
+      </DialogHeader>
+      <DialogContent width="600px">
+        {attempt.status === 'failed' && <Alert children={attempt.statusText} />}
+        <Alert
+          kind="warning"
+          primaryAction={{
+            content: 'AWS Resource Explorer',
+            href: awsResourceExplorerUrl,
+          }}
+        >
+          There may be AWS resources created by this integration that require
+          manual clean up. Visit the AWS Resource Explorer to see resources with
+          tags matching this integration.
+        </Alert>
```

Comment:
is it implied that users will be on the correct AWS account when clicking on the link? i just happened to have integrated two different aws accounts, and it could look like an empty list, if the integration we're trying to delete is not the one we are signed into?